### DISCUSSION TITLE
How to use Email Reader Plug in

### DISCUSSION THREAD

**Discussion by RazorRazer** [nextgenhealthcare/connect#5090] [Q&A]
Hey,

Someone is trying to send an attachment to an email everyday, how do I connect to this email? How do I pick up the attachment? Which one should I use for my use, IMAP or POP3?

How do I fill up the information in the Mirth for the connector? Can someone provide an example please?

![image](https://user-images.githubusercontent.com/101347093/157697143-446f6043-e40c-482a-8759-9d979cfdb6cf.png)

Thanks you, I would really appreciate this help.

**Comment by pacmano1** [comment]
Your email service provider would tell you whether or not your inbox is accessible by POP or IMAP and they would tell you server details.    Once you can retrieve the messages, consult the message viewer to see how they arrive to mirth.

If you select the "Attachments" radio button you will see the base64 attachment in the message viewer as msg.

**Reply by RazorRazer** [reply]
![image](https://user-images.githubusercontent.com/101347093/159494485-482d9ebe-a07e-41d5-9290-d89776843e20.png)

I'm getting this error:

![image](https://user-images.githubusercontent.com/101347093/159494713-07971994-d77e-4bf3-8e0a-fa29ba4e7ab4.png)

**Reply by pacmano1** [reply]
I recall IMAP access to office365 must be explicitly granted by your email administrator.

I think imap at office 365 does not need starttls. rather just ssl/tls.    Try changing that.    Also make sure the TLS cert stuff on the TLS extension is set up correctly.  

Mirth is no different in this respect than any email client.  If you can get to work in Thunderbird, Outlook (as an imap account), etc. it should work in Mirth.
